azure-spring-boot-samples/spring-data-gremlin-boot-sample/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>azure-spring-boot-samples</artifactId>
+        <groupId>com.microsoft.azure</groupId>
+        <version>0.0.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>spring-data-gremlin-boot-sample</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Spring Data Gremlin Boot Starter</name>
+    <description>Spring Boot Starter for Spring Data Gremlin</description>
+    <url>https://github.com/Microsoft/azure-spring-boot</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>spring-data-gremlin-boot-starter</artifactId>
+            <version>2.0.4-SNAPSHOT</version>
```

Comment:
Do we need the version here? Isn't the version already defined in the bom?

Reply:
because the starter not release yet. will remove this after first release.

Reply:
Sorry, my mistake, should update bom for this.